### PR TITLE
[ZT] Add Gateway proxy page

### PR DESCRIPTION
**Pull request by maxvp**
PCX-8045

### CODE REVIEW

File: content/cloudflare-one/policies/gateway/initial-setup/http.md [hunk 1]
```diff
@@ -14,10 +14,10 @@ Secure Web Gateway allows you to inspect HTTP traffic and control which websites
 
 To filter HTTP requests from a device:
 
-1. [Install the Cloudflare root certificate](/cloudflare-one/connections/connect-devices/warp/user-side-certificates/) on your device .
+1. [Install the Cloudflare root certificate](/cloudflare-one/connections/connect-devices/warp/user-side-certificates/) on your device.
 2. [Install the WARP client](/cloudflare-one/connections/connect-devices/warp/deployment/) on your device.
 3. In the WARP client Settings, log in to your organization’s [Zero Trust instance](/cloudflare-one/glossary/#team-name).
-4. Enable the Gateway proxy:
+4. Enable the [Gateway proxy](/cloudflare-one/policies/gateway/proxy/):
```
Comment (ranbel) [inline content/cloudflare-one/policies/gateway/initial-setup/http.md]: I think we have Enable the Gateway Proxy instructions on various pages throughout the docs. Should we try to single-source this or keep separate instructions for each?

File: content/cloudflare-one/policies/gateway/proxy.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+### TCP
+
+When the proxy is enabled, Gateway will always forward TCP traffic.
+
+By default, TCP connection attempts will timeout after 30 seconds and idle connections will disconnect after 8 hours.
+
+### UDP
+
+The UDP proxy forwards traffic to internal DNS resolvers. This allows Gateway to inspect all port 443 UDP traffic.
```
Comment (ranbel) [inline content/cloudflare-one/policies/gateway/proxy.md]: Doesn't UDP proxy have other use cases besides forwarding traffic to internal DNS resolvers? Also I don't understand the connection between internal DNS resolvers and port 443 traffic.

File: content/cloudflare-one/policies/gateway/proxy.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+When the UDP proxy is enabled, Gateway will force all HTTP/3 traffic to HTTP/2 to allow inspection. Otherwise, HTTP/3 traffic will bypass inspection. For more information, refer to [HTTP/3 inspection](/cloudflare-one/policies/gateway/http-policies/http3/).
```
Comment (ranbel) [inline content/cloudflare-one/policies/gateway/proxy.md:27]:
```suggestion
When the UDP proxy is enabled, Gateway will force all HTTP/3 traffic to HTTP/2 to allow inspection. For more information, refer to [HTTP/3 inspection](/cloudflare-one/policies/gateway/http-policies/http3/).
```

File: content/cloudflare-one/policies/gateway/proxy.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+{{<Aside type="warning" header="Limitation">}}Gateway cannot log or filter ICMP traffic.{{</Aside>}}
+
+## Enable the Gateway proxy
+
+1. In [Zero Trust](https://one.dash.cloudflare.com), go to **Settings** > **Network**.
+2. Enable **Proxy** for TCP.
+3. (Optional) Depending on your use case, you can enable **UDP** and/or **ICMP**.
+
+If you want to inspect HTTP traffic, we recommend you enable [**TLS decryption**](/cloudflare-one/policies/gateway/http-policies/tls-decryption/) to include HTTPS traffic.
```
Comment (ranbel) [inline content/cloudflare-one/policies/gateway/proxy.md]: The last sentence feels a bit out of place.

File: content/cloudflare-one/policies/gateway/initial-setup/http.md (hunk 1 above)
Comment (maxvp) [reply content/cloudflare-one/policies/gateway/initial-setup/http.md]: I'll link to `#enable-the-gateway-proxy` on each of those pages and add additional context as needed.

File: content/cloudflare-one/policies/gateway/proxy.md (hunk 2 above)
Comment (maxvp) [reply content/cloudflare-one/policies/gateway/proxy.md]: You're right -- revising now.

**Review by ranbel — APPROVED** [review]

**Pull request merged**